modules/microshift-greenboot-systemd-journal-data.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-greenboot-systemd-journal-data_{context}"]
+= Enabling systemd journal service data persistency
+
+The default configuration of the `systemd` journal service stores the data in the volatile `/run/log/journal` directory. To persist system logs across system starts and restarts, enable log persistence and set limits on the maximal journal data size. The following steps allow you to enable `systemd` journal service data persistency.
+
+.Procedure
+
+. Make the directory and create the configuration file by running the following commands:
++
+[source, terminal]
+----
+$ sudo mkdir -p /etc/systemd/journald.conf.d
+cat <<EOF | sudo tee /etc/systemd/journald.conf.d/microshift.conf &>/dev/null
```

Comment:
as @jeana-redhat pointed out, lines 17 and 18 are separate commands. Line 17 makes the directory and line 18 populates the file.

Reply:
Thank you.